app/presenters/digest_presenter.rb

```diff
@@ -1,6 +1,9 @@
 class DigestPresenter < NotificationPresenter
   attr_reader :current_project, :notifications, :template
 
+  # Jul 9th at 2:45pm
+  Time::DATE_FORMATS[:digest_format] = ->(time) { time.strftime "%h #{time.day.ordinalize} at %l:%M%P #{time.zone}" }
```

Comment:
Why is it better to add a global `DATE_FORMAT` here than to just do an `strftime` call in L26?

Reply:
`strftime` is the way it's been done through the application
`DATE_FORMATS` is a way we can enhance how it's being done. I like the idea of using the global formatter and initializing the formats we want in a single place. Helps spots consistency problems, makes things easier for re-use. Can be a smart default.

Reply:
I don't disagree, but if we're deciding to introduce a "consistent format across the app", a) this doesn't feel like the right PR for that; when we open the PR for that b) we should sweep the code to make sure we're adding the consistent formatting across the board, and not in this one place...

Reply:
I'll change it back.

My thinking is the debate between "Making small improvements which break consistency, or continuing doing things the same way simply because it's the way we have been doing it".
I like the idea of grand sweeping changes but they don't always end up happening.